staging/src/k8s.io/kubectl/pkg/cmd/version/version.go-e

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
Is this an accidental file?

Reply:
I don't know detail, but I think so.
I'll remove this file.

Reply:
Nice catch @ardaguclu !